src/relion/zocalo/wrapper.py

```diff
@@ -149,30 +151,21 @@ def run(self):
 
             ### Extract and send Icebreaker results as histograms if the Icebreaker grouping job has run
             if not self.opts.stop_after_ctf_estimation:
-                icebreaker_file_path = (
-                    self.working_directory / "External/Icebreaker_group/"
-                )
                 attachment_list = []
                 try:
-                    pdf_file_name = icebreaker_histogram.create_pdf_histogram(
+                    pdf_file_path = icebreaker_histogram.create_pdf_histogram(
                         self.working_directory
                     )
-                    json_file_name = icebreaker_histogram.create_json_histogram(
+                    json_file_path = icebreaker_histogram.create_json_histogram(
                         self.working_directory
                     )
-                    if pdf_file_name is None or json_file_name is None:
+                    if pdf_file_path is None or json_file_path is None:
                         pass
                     elif should_send_icebreaker:
```

Comment:
I believe this can be simplified to `if should_send_icebreaker and pdf_file_path and json_file_path:` ?